pyproject.toml

```diff
@@ -1,6 +1,16 @@
 # Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
 # See file LICENSE for terms.
 
+[tool.codespell]
```

Comment:
`pre-commit run --all-files` fails for me without this configuration. The failure is:
```
.devcontainer/README.md:3: implementors ==> implementers
```
But that's a proper name in that file, so it's not a typo.